bicep/network-new.bicep

```diff
@@ -160,108 +160,22 @@ var nsg_rules = {
     //
 
     // SSH internal rules
-    AllowSshFromJumpboxIn: ['320', 'Inbound', 'Allow', 'Tcp', 'Ssh', 'asg', 'asg-jumpbox', 'asg', 'asg-ssh']
     AllowSshFromComputeIn: ['330', 'Inbound', 'Allow', 'Tcp', 'Ssh', 'subnet', 'compute', 'asg', 'asg-ssh']
     AllowSshToComputeIn: ['360', 'Inbound', 'Allow', 'Tcp', 'Ssh', 'asg', 'asg-ssh', 'subnet', 'compute']
 
     // All communications inside compute subnet
     AllowAllComputeComputeIn: ['365', 'Inbound', 'Allow', 'Tcp', 'All', 'subnet', 'compute', 'subnet', 'compute']
 
-    // Scheduler
-    AllowSchedIn: ['369', 'Inbound', 'Allow', '*', 'Shed', 'asg', 'asg-sched', 'asg', 'asg-sched']
-    //AllowPbsClientIn            : ['370', 'Inbound', 'Allow', '*', 'Pbs', 'asg', 'asg-pbs-client', 'asg', 'asg-pbs']
-    AllowSchedComputeIn: ['380', 'Inbound', 'Allow', '*', 'Shed', 'asg', 'asg-sched', 'subnet', 'compute']
-    //      AllowComputePbsClientIn     : ['390', 'Inbound', 'Allow', '*', 'Pbs', 'subnet', 'compute', 'asg', 'asg-pbs-client']
-    AllowComputeSchedIn: ['400', 'Inbound', 'Allow', '*', 'Shed', 'subnet', 'compute', 'asg', 'asg-sched']
-    //      AllowComputeComputeSchedIn  : ['401', 'Inbound', 'Allow', '*', 'Shed', 'subnet', 'compute', 'subnet', 'compute']
-
     // CycleCloud
-    AllowCycleClientIn: [
-      '450'
-      'Inbound'
-      'Allow'
-      'Tcp'
-      'CycleCloud'
-      'asg'
-      'asg-cyclecloud-client'
-      'asg'
-      'asg-cyclecloud'
-    ]
-    AllowCycleClientComputeIn: [
-      '460'
-      'Inbound'
-      'Allow'
-      'Tcp'
-      'CycleCloud'
-      'subnet'
-      'compute'
-      'asg'
-      'asg-cyclecloud'
-    ]
-    AllowCycleServerIn: [
-      '465'
-      'Inbound'
-      'Allow'
-      'Tcp'
-      'CycleCloud'
-      'asg'
-      'asg-cyclecloud'
-      'asg'
-      'asg-cyclecloud-client'
-    ]
+    AllowCycleClientComputeIn: ['460', 'Inbound', 'Allow', 'Tcp', 'CycleCloud', 'subnet', 'compute', 'asg', 'asg-cyclecloud']
 
     // Deny all remaining traffic

```

Comment:
Can we update this comment about why we chose to comment out the deny?

Reply:
I will review all rules